website_and_docs/content/blog/2022/introducing-selenium-manager.md

```diff
@@ -49,12 +49,12 @@ ecosystem emerged. Such as: [WebDriverManager](https://bonigarcia.dev/webdriverm
 [WebDriverManager.Net](https://github.com/rosolko/WebDriverManager.Net) for C#. 
 
 All these projects served as an 
-[inspiration]({{< ref "../../documentation/webdriver/getting_started/install_drivers.md#2-driver-management-software" >}}) 
+[inspiration]({{< ref "../../documentation/webdriver/troubleshooting/errors/driver_location.md#2-driver-management-libraries" >}}) 
```

Comment:
The link should have been `driver_location/#driver-management-libraries`

Reply:
Not sure, I think it is better to give the whole overview and let the use choose.